libs/pages/services/src/lib/page-services.tsx

```diff
@@ -86,22 +86,39 @@ export function PageServices() {
           })
         ),
     },
-    {
-      name: 'delete',
-      action: () =>
-        dispatch(
-          deleteEnvironmentActionsCancelDeployment({
-            projectId,
-            environmentId,
-            withDeployments:
-              location.pathname === SERVICES_URL(organizationId, projectId, environmentId) + SERVICES_DEPLOYMENTS_URL,
-          })
-        ),
-    },
+    // {
+    //   name: 'delete',
+    //   action: () =>
+    //     dispatch(
+    //       deleteEnvironmentActionsCancelDeployment({
+    //         projectId,
+    //         environmentId,
+    //         withDeployments:
+    //           location.pathname === SERVICES_URL(organizationId, projectId, environmentId) + SERVICES_DEPLOYMENTS_URL,
+    //       })
+    //     ),
+    // },
   ]
 
+  const removeEnvironment = () => {
+    dispatch(
+      deleteEnvironmentActionsCancelDeployment({
```

Comment:
We can directly call this function deleteEnvironment ?

And we don't need to refetch `withDeployments`, because the environment is deleted and we didn't have services now. 
I think we need a redirect to the list of the Environments